### PR TITLE
Improving output of pretty formatter on CI

### PR DESCRIPTION
**Pull request by hferentschik**
Fix for issue #419 

Also cleaning up Cucumber config in general and speeding up tests.

### CODE REVIEW

**Comment by hferentschik** [comment]
@budhrg check out the build results for this pull request. I think things are clearer now. Unfortunately, there is something wrong atm. The cdk box tests are not run.

**Comment by hferentschik** [comment]
test this please

**Comment by hferentschik** [comment]
> Unfortunately, there is something wrong atm. The cdk box tests are not run.

Seems 'cdk' got removed in the Jenkins configuration!?

**Comment by hferentschik** [comment]
test this please

**Comment by brgnepal** [comment]
> Seems 'cdk' got removed in the Jenkins configuration!?

It is there. 

```
build_cmd += 'bundle exec rake features CUCUMBER_OPTS=\'-p ci\' PROVIDER=libvirt BOX=adb,cdk && '
```

File: features/kubernetes.feature [hunk 1]
```diff
@@ -42,3 +42,5 @@ Feature: Command output from various Kubernetes related commands
       | box   | provider   | ip          |
       | adb   | virtualbox | 10.10.10.42 |
       | cdk   | virtualbox | 10.10.10.42 |
+      | adb   | libvirt    | 10.10.10.42 |
+      | cdk   | libvirt    | 10.10.10.42 |
```
Comment (brgnepal) [inline features/kubernetes.feature:46]: :+1:

**Comment by hferentschik** [comment]
> It is there. 

I added it again

**Comment by hferentschik** [comment]
Reverting back to CDK 2.1 to see whether it makes a difference...

**Comment by hferentschik** [comment]
test this please

**Comment by hferentschik** [comment]
All tests pass with CDK 2.1. There is a regression in 2.2 which seems to be connected to the availability of OpenShift after 'vagrant up'.

**Comment by hferentschik** [comment]
test this please

**Comment by hferentschik** [comment]
Added a @todo annotation for now to the CDK OpenShift test and disabled it from _default_ and _ci_ profile. This way we can get the update to CDK 2.3 as well as the improved reporting while we figure out issue #415.

@budhrg WDYT?

**Comment by hferentschik** [comment]
Closing this pull request. Will be superseded by an updated, slightly differently structured pull request.

**Pull request closed without merging**